abb_irb120_moveit_config/config/kinematics.yaml

```diff
@@ -0,0 +1,6 @@
+manipulator:
+  #kinematics_solver: abb_irb120_manipulator_kinematics/IKFastKinematicsPlugin
+  kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
+  kinematics_solver_search_resolution: 0.005
+  kinematics_solver_timeout: 0.005
+  kinematics_solver_attempts: 3
```

Comment:
(optional) this moveit cfg has no ikfast plugin. We may want to log an issue for that.